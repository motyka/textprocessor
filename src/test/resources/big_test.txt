Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla vulputate velit vel faucibus sagittis. In tempor urna a rutrum mollis. Nulla ac orci nec purus iaculis mollis. Morbi elementum facilisis nisl. Maecenas orci purus, tristique quis ornare eu, ullamcorper non sapien. In hac habitasse platea dictumst. Maecenas eu felis blandit, blandit arcu in, dignissim nibh. Integer venenatis elementum hendrerit. Pellentesque tempor maximus ipsum quis imperdiet. Donec suscipit viverra neque vitae iaculis. Praesent porttitor tortor ut pellentesque tempor.

Mauris rutrum sollicitudin felis, in posuere odio iaculis sit amet. Cras congue porttitor enim, faucibus efficitur ligula fringilla vel. Phasellus erat ante, eleifend a felis in, condimentum semper magna. Curabitur a felis in lacus cursus pharetra. Phasellus at faucibus neque. Nunc tincidunt nisi vitae dui tincidunt posuere. Maecenas malesuada mi non lorem euismod hendrerit. Mauris est dui, tristique sed rhoncus ut, ultricies eget quam. Morbi rutrum erat a aliquam sollicitudin. Pellentesque luctus sed elit a posuere. Aenean ac nulla nunc. Nulla sollicitudin hendrerit mi, ut vestibulum mauris dapibus sit amet. Interdum et malesuada fames ac ante ipsum primis in faucibus.

Aliquam erat volutpat. Cras tortor nisi, volutpat at nisi a, consectetur fermentum arcu. Mauris lacinia, mauris ut aliquam suscipit, tellus ante euismod mi, sed finibus dui nunc non ex. Nunc metus nisi, consequat at laoreet id, aliquet eget arcu. Nulla facilisi. Morbi ac arcu leo. Morbi accumsan nec elit ut sagittis. Morbi pharetra viverra justo quis vehicula. Vestibulum interdum feugiat velit id sodales.

Sed vel pellentesque lacus, sit amet varius turpis. Donec sed arcu congue, rhoncus nisi finibus, aliquet sapien. Integer elementum nibh porttitor magna vehicula euismod. Morbi facilisis eros et sapien elementum, vel tempus arcu sodales. Nunc eu lectus urna. Nam dictum odio sem, ac luctus orci finibus sed. Phasellus quis maximus nunc. Vestibulum sodales metus lacus, tristique mollis purus viverra non. Proin id augue sed velit ultricies elementum laoreet eu erat. Etiam quis ipsum id nunc consequat sodales.

Donec rutrum ullamcorper tortor, a fermentum magna dictum a. Curabitur commodo urna quis magna vestibulum luctus. Fusce fringilla mauris id lectus aliquam luctus. Ut iaculis, mauris sit amet sollicitudin laoreet, nisl purus aliquam eros, in gravida odio urna eu augue. Nulla placerat elit eros, vitae gravida nunc hendrerit ac. Sed dolor magna, rutrum sed hendrerit vel, commodo a mi. Sed eu lorem ultrices, vehicula enim at, interdum neque. Proin vehicula ac nulla sed pulvinar. Proin mollis ante efficitur porta consectetur. Morbi viverra molestie libero, non gravida quam sodales eget. Pellentesque odio tellus, faucibus eu nisl maximus, mollis sagittis mauris. Pellentesque sagittis lacus nec quam posuere semper. Quisque vulputate augue nec leo consectetur molestie. Donec pulvinar, urna nec elementum placerat, augue ante viverra sapien, vitae eleifend ligula nisl in turpis.