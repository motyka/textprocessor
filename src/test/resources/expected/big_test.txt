[{text=Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla vulputate velit vel faucibus sagittis. In tempor urna a rutrum mollis. Nulla ac orci nec purus iaculis mollis. Morbi elementum facilisis nisl. Maecenas orci purus, tristique quis ornare eu, ullamcorper non sapien. In hac habitasse platea dictumst. Maecenas eu felis blandit, blandit arcu in, dignissim nibh. Integer venenatis elementum hendrerit. Pellentesque tempor maximus ipsum quis imperdiet. Donec suscipit viverra neque vitae iaculis. Praesent porttitor tortor ut pellentesque tempor., start=0, length=553, contains=true}, {text=Mauris rutrum sollicitudin felis, in posuere odio iaculis sit amet. Cras congue porttitor enim, faucibus efficitur ligula fringilla vel. Phasellus erat ante, eleifend a felis in, condimentum semper magna. Curabitur a felis in lacus cursus pharetra. Phasellus at faucibus neque. Nunc tincidunt nisi vitae dui tincidunt posuere. Maecenas malesuada mi non lorem euismod hendrerit. Mauris est dui, tristique sed rhoncus ut, ultricies eget quam. Morbi rutrum erat a aliquam sollicitudin. Pellentesque luctus sed elit a posuere. Aenean ac nulla nunc. Nulla sollicitudin hendrerit mi, ut vestibulum mauris dapibus sit amet. Interdum et malesuada fames ac ante ipsum primis in faucibus., start=557, length=678, contains=true}, {text=Aliquam erat volutpat. Cras tortor nisi, volutpat at nisi a, consectetur fermentum arcu. Mauris lacinia, mauris ut aliquam suscipit, tellus ante euismod mi, sed finibus dui nunc non ex. Nunc metus nisi, consequat at laoreet id, aliquet eget arcu. Nulla facilisi. Morbi ac arcu leo. Morbi accumsan nec elit ut sagittis. Morbi pharetra viverra justo quis vehicula. Vestibulum interdum feugiat velit id sodales., start=1239, length=408, contains=false}, {text=Sed vel pellentesque lacus, sit amet varius turpis. Donec sed arcu congue, rhoncus nisi finibus, aliquet sapien. Integer elementum nibh porttitor magna vehicula euismod. Morbi facilisis eros et sapien elementum, vel tempus arcu sodales. Nunc eu lectus urna. Nam dictum odio sem, ac luctus orci finibus sed. Phasellus quis maximus nunc. Vestibulum sodales metus lacus, tristique mollis purus viverra non. Proin id augue sed velit ultricies elementum laoreet eu erat. Etiam quis ipsum id nunc consequat sodales., start=1651, length=509, contains=true}, {text=Donec rutrum ullamcorper tortor, a fermentum magna dictum a. Curabitur commodo urna quis magna vestibulum luctus. Fusce fringilla mauris id lectus aliquam luctus. Ut iaculis, mauris sit amet sollicitudin laoreet, nisl purus aliquam eros, in gravida odio urna eu augue. Nulla placerat elit eros, vitae gravida nunc hendrerit ac. Sed dolor magna, rutrum sed hendrerit vel, commodo a mi. Sed eu lorem ultrices, vehicula enim at, interdum neque. Proin vehicula ac nulla sed pulvinar. Proin mollis ante efficitur porta consectetur. Morbi viverra molestie libero, non gravida quam sodales eget. Pellentesque odio tellus, faucibus eu nisl maximus, mollis sagittis mauris. Pellentesque sagittis lacus nec quam posuere semper. Quisque vulputate augue nec leo consectetur molestie. Donec pulvinar, urna nec elementum placerat, augue ante viverra sapien, vitae eleifend ligula nisl in turpis., start=2164, length=881, contains=false}]